host/src/getparameters.cpp

```diff
@@ -810,6 +811,19 @@ void get_commandpars(const int* argc,
 				printf("Warning: value of -rmstol argument ignored. Value must be a double greater than 0.\n");
 		}
 
+		// Argument: choose wether to output xml or not
+		//If the value is 1, xml output will still be generated
+		if (strcmp("-xmloutput", argv [i]) == 0)
+		{
+			arg_recognized = 1;
+			sscanf(argv [i+1], "%ld", &tempint);
+			
+			if (tempint == true)
```

Comment:
It would be easier to read if the statement just tests the negative case:
`if (tempint == 0)`
`    mypars->output_xml = false;`
`else`
`   mypars->output_xml = true;`